### ISSUE TITLE
Bar Chart default color accesor

### ISSUE THREAD

**Issue by yang-wei** [esbullington/react-d3#241]
Currently the bar chart is:

![image](https://cloud.githubusercontent.com/assets/5494874/8889581/c670a8c8-331a-11e5-8e70-09545d1a70f4.png)

Before

![image](https://cloud.githubusercontent.com/assets/5494874/8889580/a674d742-331a-11e5-8559-0385a0d2988d.png)

I think expected color is the before one.

**Comment by yang-wei** [comment]
@MatthewHerbst Any thought on this ?

**Comment by MatthewHerbst** [comment]
I agree. I could hot fix this now if you want, or maybe if we can figure out #237 then I can do the entire overhaul.

**Comment by yang-wei** [comment]
@MatthewHerbst Can we fix this first because I want to update our npm registry fast. Thanks.

**Comment by yang-wei** [comment]
I found it's not good to tweak the default colorAccesor property of BarChart so I think the best way to do this now it to turn BarChart into multi series chart.

If we pass it one series it will looks like the before.

Probably I can do this on tomorrow.

**Comment by MatthewHerbst** [comment]
I was going to try doing this tonight:

```javascript
// Recall that d3.scale.category20c() actually returns the function d3.scale.ordinal()
// category20c() is just short-hand for d3.scale.ordinal().range(d3_category20c);
colors: (colorAccessorValue) => (x) => '#3182bd'
```

But the tests break when I do that. Can't figure out when

**Comment by yang-wei** [comment]
Another way to solve this (which I think is better) is to make BarChart into multi series graph.

If we pass in one series, it will become what show in our website.

**Comment by MatthewHerbst** [comment]
The cool thing about making BarChart me multi-series would also be that we could allow for stacked bar charts. I think that would be awesome. In fact, you can really think of all BarCharts as being stacked bar charts.

**Comment by yang-wei** [comment]
>  stacked bar charts
Working on this

**Comment by MatthewHerbst** [comment]
Hey @yang-wei Let me know when you've got this ready. I'm holding off on making a pr for #237 until you make these changes since I don't want to have to do any crazy merge-conflicts.

**Comment by yang-wei** [comment]
@MatthewHerbst  This is merged in #245. Please go ahead !
